src/QuickMarcEditor/useSaveRecord/useSubmitRecord/useSumbitRecord.js

```diff
@@ -96,13 +101,13 @@ const useSubmitRecord = ({
       [MARC_TYPES.HOLDINGS]: `${basePath}/edit-holdings/${externalId}`,
     };
 
-    await refreshPageData(fieldIds, QUICK_MARC_ACTIONS.EDIT, externalId);
-
-    history.push({
+    await history.push({
```

Comment:
It is also needed to change URL during derive, otherwise there will be `shared=true` for the created local record.